### PR TITLE
Update information about clisp unit testing #180

### PR DESCRIPTION
**Pull request by mappuji**
Pull request for fixing this issue https://github.com/exercism/common-lisp/issues/180

In this PR I add update for how to do unit test in clisp with quickload.

### CODE REVIEW

**Comment by verdammelt** [comment]
Thank you! I want to do/propose a few edits because if we give explicit instructions like this for clisp I'd like to also mention sbcl (our other main implementation that people use...)

File: docs/TESTS.md [hunk 1]
```diff
@@ -9,18 +9,33 @@ with `(lisp-unit:run-tests :all :point-mutations-test)`.
 
 ## Making your first Common Lisp solution
 
-To create lisp code that can be loaded with `(load "dna")`
-for the first exercise, put this code in `dna.lisp`:
+To create lisp code that can be loaded with `#-xlisp-test (load "hamming")`
```
Comment (verdammelt) [inline docs/TESTS.md]: The `#-xlisp-test` is a feature check. This is not needed by a user when loading the file. remove that from this change.

File: docs/TESTS.md [hunk 2]
```diff
@@ -9,18 +9,33 @@ with `(lisp-unit:run-tests :all :point-mutations-test)`.
...
+(in-package #:hamming)
 
-(defun hamming-distance (dna1 dna2)
-  "Determine number of mutations between DNA strands by computing the Hamming Distance."
+(defun distance (dna1 dna2)
+  "Number of positional differences in two equal length dna strands."
+  ;;; your solution here
   )
 ```
+
```
Comment (verdammelt) [inline docs/TESTS.md]: Let's make a new header here. Maybe called "How to run the tests". Then we can say how different implementations have different details and that here is how to do it for SBCL and Clisp (your instructions here for Clisp). How to do it for SBCL is: `sbcl --load hamming-test.lisp --quit`

**Comment by verdammelt** [comment]
@mappuji I can only update this PR if I push to your branch, which I'm assuming I cannot. So I ended up creating a new PR with your change and my update #185.

Can you review my proposed edit?

**Comment by mappuji** [comment]
Moved to #185

**Pull request closed without merging**